applications/backend/main.go

```diff
@@ -33,13 +38,24 @@ func main() {
 			ReturnedAt: time.Date(2025, 4, 16, 17, 42, 0, 0, time.UTC),
 		},
 	}
+
+	requestDurations := httpconv.NewServerRequestDuration()
+	prometheus.Register(requestDurations)
+	recordRequestDuration := func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			start := time.Now()
+			next.ServeHTTP(w, r)
+			requestDurations.With(httpconv.RequestMethodAttr(r.Method), httpconv.UrlSchemeAttr("http"), httpconv.RouteAttr(r.Pattern)).Observe(float64(time.Since(start).Seconds()))
+		})
+	}
```

Comment:
Example of library usage

First two attributes are required. Users can then pass arbitrary optional args but they must be allowed on the specific metric